coverage/coverage-final.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I'm sure it doesn't matter, but is this document doing anything?

Reply:
It is related to testing.  It is in my .gitignore, but I think I need to delete for the files to not show up